jobs/transforms/exp_model_audits_test.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for jobs.transforms.exp_model_audits."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.platform import models
+from jobs import job_test_utils
+from jobs.transforms import exp_model_audits
+
+import apache_beam as beam
+
+(base_models, exp_models) = models.Registry.import_models(
+    [models.NAMES.base_model, models.NAMES.exploration])
+
+
+class ValidateExplorationSnapshotMetadataModelCommitCmdsSchemaTests(
```

Comment:
Please add more tests specific to the exploration change.

Reply:
@vojtechjelinek, made some changes, PTAL

Reply:
Looks better but we don't want to just test the exploration change separately but together with the `ValidateExplorationCommitCmdsSchema`. Right now you're just testing it seprately.

Reply:
@vojtechjelinek making sure,  
so then I will have to test it by making `exp_models.ExplorationSnapshotMetadataModel(...values)` and passing it to beam func right?
example:
```python
    def test_validate_exp_model_object_with_invalid_cmd(self):
        invalid_commit_cmd_model = exp_models.ExplorationSnapshotMetadataModel(
            id='model_id-1',
            created_on=self.YEAR_AGO,
            last_updated=self.NOW,
            committer_id='committer_id',
            commit_type='create',
            commit_cmds_user_ids=[
                'commit_cmds_user_1_id', 'commit_cmds_user_2_id'],
            content_user_ids=['content_user_1_id', 'content_user_2_id'],
            commit_cmds=[{'invalid': 'data'}])

        output = (
            self.pipeline
            | beam.Create([invalid_commit_cmd_model])
            | beam.ParDo(
                exp_model_audits
                .ValidateExplorationCommitCmdsSchema())
        )

        self.assert_pcoll_equal(output, [
                        audit_errors.CommitCmdsValidateError(
                invalid_commit_cmd_model,
                '{u\'invalid\': u\'data\'}',
                'Missing cmd key in change dict')
        ])
```